### DISCUSSION TITLE
pubic rendezvous_server down starting Feb12 morning other options

### DISCUSSION THREAD

**Discussion by johnh530** [rustdesk/rustdesk#3186] [General]
I have had only marginal success on connecting with the general rendezvous_server today.

I could spin up a EC2 server and install a private server - but I only need to do about 15 minutes of work on a project.

Is there a list of contacts which would be able to be the rendezvous_server for a short period of fime?

Are others experiencing issues with the general rendezvous_servers?

Are these servers likely to be back in operation soon?

Country US
version 1.1.8 ( mac os is broken on 1.1.9 and nightly build related to keyboard working).

johnh...

**Comment by rustdesk** [comment]
I checked US server, it is up. But we do have limit bandwidth.

**Reply by johnh530** [reply]
Was able to get in a few minutes ago - but tried numerous times today.
Thanks for checking.

**Comment by johnh530** [comment]
Could you check to see if the US rendezvous_servers are up - have not been able to get in for the last hour?

johnh...

**Reply by rustdesk** [reply]
It's up, but it is a cheap vultr VPC, performance is not guarented.

**Comment by johnh530** [comment]
When you say "performance is not guaranteed" - does that mean the one can expect operation though there might be a lag in performance or that it is possible that no operation is possible?

johnh...
________________________________
